File: C++ for Everyone, 2nd Ed, Horstmann/Chapter4/P4-3B.cpp
//Write programs that read a line of input as a string and print
//b.Every second letter of the string.

#include<iostream>

using namespace std;

int main()
{
    cout << "Enter a string: ";
    string input;
    getline(cin, input);

    for(int i =0; i < input.length(); i += 2)
    {
        cout << input.substr(i, 1);
    }

    return 0;
}

File: Python-For-Everyone-Horstmann/Chapter4-Loops/P4.3B.py
# Write programs that read a line of input as a string and print
#     a.  Only the uppercase letters in the string.
#     b.  Every second letter of the string.
#     c.  The string, with all vowels replaced by an underscore.
#     d.  The number of digits in the string.
#     e.  The positions of all vowels in the string.

inputString = str(input("Enter a string: "))
result = ""

for i in range(0, len(inputString), 2):
    result += inputString[i]

print(result)